### PR TITLE
docs: expand closeto example 

### PR DESCRIPTION
**Pull request by TheChronicMonster**
Added to the closeTo example per [this](https://discord.com/channels/628402598663290882/749639931399241792/923230337390215218) Discord discussion.

### CODE REVIEW

File: docs/src/rsh/step/index.md [hunk 1]
```diff
@@ -464,6 +464,22 @@ closeTo( Who, after, nonNetPayAmt )
 ```
 
 @{ref("rsh", "closeTo")} Has participant `{!rsh} Anybody` make a publication, then transfer the `{!rsh} balance()` and the non-network pay amount to `{!rsh} Who` and end the DApp after executing the function `{!rsh} after` in a step.
+
+As an example, codifying the prose above would look like:
+
+```reach
+// Has participant Who make a publication
+  Who.publish();
```
Comment (jeapostrophe) [inline docs/src/rsh/step/index.md:472]: `Anybody`, not `Who`

Comment (jeapostrophe) [reply docs/src/rsh/step/index.md:472]: Also, I don't like that the comments don't line up with the code

File: docs/src/rsh/step/index.md [hunk 2]
```diff
@@ -464,6 +464,22 @@ closeTo( Who, after, nonNetPayAmt )
...
+// transfer the balance() and the non-network pay amount to Who
+  transfer([ balance(), ...nonNetPayAmt ]).to(Who);
+  commit();
+// end the DApp after executing the function after in a step
+  after();
+  exit();
+```
+
+By comparing this example to `closeTo`, you can see that using `closeTo` helps to save additional lines of code.
```
Comment (jeapostrophe) [inline docs/src/rsh/step/index.md:481]: You should use `{!rsh} closeTo`. And it's not just about saving typing, it's about saving thinking.

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: docs/src/rsh/step/index.md (hunk 1 above)
Comment (TheChronicMonster) [reply docs/src/rsh/step/index.md:472]: In this example, we're talking about this participant, `Who`. "Who" is publishing and "Who" is receiving a payment transfer. Would it be confusing to refer to "Anybody?"
Perhaps it's better to wrap "Who" in monotype in the comment?
// Has participant `Who` make a publication

Comment (jeapostrophe) [reply docs/src/rsh/step/index.md:472]: My point is that `Who`, this participant, is *not* the one publishing. `Anybody` is publishing. (I.e. your code sample is wrong)

File: docs/src/rsh/step/index.md [hunk 3]
```diff
@@ -463,7 +463,23 @@ It accepts an optional bytes argument, which is included in any reported violati
 closeTo( Who, after, nonNetPayAmt )
 ```
 
-@{ref("rsh", "closeTo")} Has participant `{!rsh} Anybody` make a publication, then transfer the `{!rsh} balance()` and the non-network pay amount to `{!rsh} Who` and end the DApp after executing the function `{!rsh} after` in a step.
+@{ref("rsh", "closeTo")} Has participant `{!rsh} Anybody` make a publication, then transfer the `{!rsh} balance()` and the non-network pay amount to `{!rsh} Who` and end the dApp after executing the function `{!rsh} after` in a step.
```
Comment (jeapostrophe) [inline docs/src/rsh/step/index.md:466]: :( We say "DApp" all over the place, so if you want to try and convince me to change that, you'll have to do that separately than this

File: docs/src/rsh/step/index.md [hunk 4]
```diff
@@ -463,7 +463,23 @@ It accepts an optional bytes argument, which is included in any reported violati
...
+
+As an example, codifying the prose above would look like:
+
+```reach
+// Has participant `{!rsh} Anybody` make a publication
```
Comment (jeapostrophe) [inline docs/src/rsh/step/index.md:471]: Does this render properly? I assume it doesn't

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: docs/src/rsh/step/index.md (hunk 3 above)
Comment (TheChronicMonster) [reply docs/src/rsh/step/index.md:466]: That's not a hill I'm willing to die on at this point in time.

File: docs/src/rsh/step/index.md (hunk 4 above)
Comment (TheChronicMonster) [reply docs/src/rsh/step/index.md:471]: Ticks don't render in comments. We can use single quotes to indicate signification in comments. e.g.
// Has participant 'Anybody' make a publication
// Transfer the balance() and the non-network payment amount to 'Who'
// End the DApp after executing the function 'after' in a step

Comment (jeapostrophe) [reply docs/src/rsh/step/index.md:471]: Sounds good

**Review by jeapostrophe — APPROVED** [review]

**Pull request merged**